skyhook.agent.nut

```diff
@@ -4,31 +4,27 @@
  *
  * @author Satya Yarramsetty <satya.yarramsetty@trueposition.com>
  *
- * @version 1.0.0
+ * @version 2.0.0
  */
 
 class skyhook {
```

Comment:
Class names should be UpperCamelCase https://electricimp.com/docs/squirrel/codestyle/#class-names